### PR TITLE
c-ares files

### PR DESCRIPTION
**Pull request by JaiXJM-BB**

### CODE REVIEW

**Comment by JaiXJM-BB** [comment]
Marking as good to merge for visibility and review purposes; my understanding is that the curl team may take these files over.

File: .github/workflows/c-ares.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  build:
+    runs-on: build-files-runner
```
Comment (chachoi) [inline .github/workflows/c-ares.yml:14]: Need to change this to self-hosted

**Comment by chachoi** [comment]
Do we know why c-ares workflow is not being triggered?

**Comment by JaiXJM-BB** [comment]
> Do we know why c-ares workflow is not being triggered?

I believe it is manually disabled at the moment.

**Comment by chachoi** [comment]
> > Do we know why c-ares workflow is not being triggered?
> 
> I believe it is manually disabled at the moment.

I enabled it now. I forgot I disabled it :)

**Review by chachoi — APPROVED** [review]
Changes look good to me. I see that the last action build succeeded, so I'm assuming the build is fine.

**Pull request merged**